### PR TITLE
build(all): upgrade to lerna 6

### PR DESCRIPTION
**Pull request by hanmangokiwi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/develop/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Building with Lerna does not build in parallel nor take advantage of caching

## What is the new behavior?

Building with Lerna builds in parallel and takes advantage of caching

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: tools/dgeni/package.json [hunk 1]
```diff
@@ -10,28 +10,13 @@
...
-  "devDependencies": {
-    "@daffodil/auth": "0.0.0-PLACEHOLDER",
-    "@daffodil/authorizenet": "0.0.0-PLACEHOLDER",
-    "@daffodil/branding": "0.0.0-PLACEHOLDER",
-    "@daffodil/cart": "0.0.0-PLACEHOLDER",
-    "@daffodil/category": "0.0.0-PLACEHOLDER",
-    "@daffodil/checkout": "0.0.0-PLACEHOLDER",
-    "@daffodil/contact": "0.0.0-PLACEHOLDER",
-    "@daffodil/core": "0.0.0-PLACEHOLDER",
-    "@daffodil/docs-utils": "0.0.0-PLACEHOLDER",
```
Comment (damienwebdev) [inline tools/dgeni/package.json:22]: Looks like `@daffodil/docs-utils` is a real dependency.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

**Comment by damienwebdev** [comment]
Looks great!

**Review by damienwebdev — APPROVED** [review]

**Comment by damienwebdev** [comment]
Needs review by @griest024

**Review by griest024 — APPROVED** [review]
<3 Big win!!

**Pull request merged**